src/errors.rs

```diff
@@ -37,6 +37,8 @@ pub enum Error {
     /// `Event::BytesDecl` must start with *version* attribute. Contains the attribute
     /// that was found or `None` if an xml declaration doesn't contain attributes.
     XmlDeclWithoutVersion(Option<String>),
+    /// Empty `Event::DocType`.
+    EmptyDocType,
```

Comment:
I suggest to expand description a bit by showing an example of incorrect and correct declarations:
- `<!doctype>` is incorrect
- `<!doctype >` is correct
- `<!doctype sometype>` is correct

Reply:
Thank you! Done.